### PR TITLE
[CLI-600] Allow 'appc use latest' offline

### PR DESCRIPTION
**Pull request by garymathews**
- Using ``appc use latest``` offline sets the active version as the latest CLI version installed

[JIRA Ticket](https://jira.appcelerator.org/browse/CLI-600)

### CODE REVIEW

File: lib/use.js [hunk 1]
```diff
@@ -30,10 +30,20 @@ function use(opts, callback, wantVersion) {
 				// looks like we are offline
 				if (err.code === 'ENOTFOUND' || 'ENOENT') {
 					var versions = util.getInstalledVersions();
+					// set active version as latest installed version
+					if (getLatest) {
+						latest = versions[0];
+						installBin = util.getInstallBinary(opts, latest);
```
Comment (ingo) [inline lib/use.js]: Should we declare installBin? Is it a global?

Comment (garymathews) [inline lib/use.js]: yes it's global, https://github.com/appcelerator/appc-install/blob/master/lib/use.js#L78

Comment (jhaynie) [inline lib/use.js]: it shouldn't be global and should be declared. this is a memory leak.

Comment (garymathews) [inline lib/use.js]: I fixed this as part of https://github.com/appcelerator/appc-install/pull/18

**Pull request merged**